### PR TITLE
New Donor Items, minor tweaks.

### PR DESCRIPTION
**Pull request by Ashvor**
:cl: Ashvor
rscadd: Added new donor armor and weapon skins for Donators.
tweak: Knife may now be stored in Marine EVA boots.
tweak: SOE combat gloves armor removed. Due to SOE Space/Normal armor already providing armor.
/:cl:

### CODE REVIEW

File: code/modules/halo/clothing/odst.dm [hunk 1]
```diff
@@ -66,7 +66,7 @@
 	armor = list(melee = 55, bullet = 50, laser = 55, energy = 45, bomb = 40, bio = 100, rad = 25)
 	//specials = list(/datum/armourspecials/internal_air_tank/human) This line is disabled untill a dev can fix the internals code for it.
 	item_flags = STOPPRESSUREDAMAGE|THICKMATERIAL
-	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS | LEGS
+	body_parts_covered = UPPER_TORSO | LOWER_TORSO | ARMS | LEGS | FEET | HANDS
```
Comment (X0-11) [inline code/modules/halo/clothing/odst.dm]: Gloves already have armour and armour thickness. Boots also have armour. Leave their jobs to them.

**Review by X0-11 — DISMISSED** [review]

File: code/modules/halo/clothing/odst.dm (hunk 1 above)
Comment (Ashvor) [reply code/modules/halo/clothing/odst.dm]: SOE suit applies to the feet and hands. SOE Space armor already have armor to those locations. I'm just putting up a balance suggestion to keep it fair.

Comment (X0-11) [reply code/modules/halo/clothing/odst.dm]: Oversight. Remove the armour from the soe's ones. Likely a copypaste issue

Comment (Ashvor) [reply code/modules/halo/clothing/odst.dm]: Sorry misread. Will remove the armor from SOE gloves and boots. Since the suit covers those parts.

Comment (X0-11) [reply code/modules/halo/clothing/odst.dm]: Yee

Comment (X0-11) [reply code/modules/halo/clothing/odst.dm]: Looks like I misunderstood your response: remove the coverage of soe armour of hands and feet

**Review by BDpuffy420 — APPROVED** [review]

**Pull request merged**